### PR TITLE
(DPS-2160) Version scripts

### PR DESCRIPTION
**Pull request by jakubsikora**
## 🚀 Links

- [Feature branch](https://developers.labs.livechat.com/docs/feature/DPS-2160)
- [Jira](https://livechatinc.atlassian.net/browse/DPS-2160)

## 📓 Description
2 scripts:

1. `make-stable` e.g. `npm run version -- --type=make-stable --newVersion=3.3 --exclude=customer-chat-api` (make version 3.3 stable, current stable version will become legacy) 
Options:
`--group`: (default: DEFAULT), for chat widget use `--group=chat-widget`
`--exclude`: e.g. `--exclude=agent-chat-api,customer-chat-api`  

2. `new` e.g. `npm run version -- --type=new --newVersion=3.5 --fromVersion=3.1` (create new version 3.5, based on files from 3.1)
Options:
`--group`: (default: DEFAULT), for chat widget use `--group=chat-widget`
`--exclude`: e.g. `--exclude=agent-chat-api,customer-chat-api`  

### Features (for the website)

- New feature

### CODE REVIEW

File: scripts/version.js [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+if (argv.type) {
+  switch (argv.type) {
+    case "make-stable": {
+      if (argv.fromVersion && argv.toVersion) {
+        const silentState = shell.config.silent; // save old silent state
+        shell.config.silent = true;
+        const fromVersion = argv.fromVersion;
+        const toVersion = argv.toVersion;
+
+        // configuration-api
```
Comment (ale-kacperczyk) [inline scripts/version.js:13]: we would need support for Chat Widget (extending-chat-widget category). they have different versions and don't have rtm-reference/

File: scripts/version.js [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+        process.stdout.write("Moving configuration-api...");
+
+        shell.mkdir(`./content/management/configuration-api/v${fromVersion}`);
+        shell.exec(
+          `git mv ./content/management/configuration-api/index.mdx ./content/management/configuration-api/v${fromVersion}`
+        );
+        shell.exec(
+          `git mv ./content/management/configuration-api/v${toVersion}/index.mdx ./content/management/configuration-api/`
+        );
```
Comment (ale-kacperczyk) [inline scripts/version.js:22]: Forgot to add `shell.rm("-R", `./content/management/configuration-api/v${toVersion}`);`

File: scripts/version.js [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+
+        shell.config.silent = silentState;
+      } else {
+        console.log("--fromVersion and --toVersion are required");
+        shell.exit(1);
+      }
+      break;
+    }
+
+    case "new": {
```
Comment (ale-kacperczyk) [inline scripts/version.js:80]: We could add two messages for this case:
 "Remember to update apiVersion in the document's meta" 
"Remember to update ALL_VERSIONS in constants"

**Comment by jakubsikora** [comment]
Fixed the comments, add more options (see description)

**Pull request merged**